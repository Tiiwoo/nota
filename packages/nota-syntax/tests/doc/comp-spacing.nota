@h1: a

@h2: b