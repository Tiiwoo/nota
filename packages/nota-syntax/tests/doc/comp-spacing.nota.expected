[
  el(
    "h1",
    {
      block: true,
    },
    "a"
  ),
  el(
    "h2",
    {
      block: true,
    },
    "b"
  ),
];